src/content/docs/en/pages/devtools/cli/azion-cli-alpha/link-project/link.mdx

```diff
@@ -1,12 +1,16 @@
 ---
-title: Azion CLI's link command
+title: Azion CLI link
 description: Linking an existing project to run on the Azion Edge platform.
 meta_tags: cli, next, javascript, astro, vue, react, edge computing
 namespace: documentation_cli_link
 menu_namespace: cliMenuAlpha
 permalink: /documentation/products/cli/link-command/
 ---
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```